hack/update-codegen.sh

```diff
@@ -25,7 +25,7 @@ CODEGEN_PKG=$($GO_CMD list -m -f "{{.Dir}}" k8s.io/code-generator)
 cd $(dirname ${BASH_SOURCE[0]})/..
 
 chmod +x "${CODEGEN_PKG}/generate-internal-groups.sh"
-source "${CODEGEN_PKG}/generate-groups.sh" \
+source "${CODEGEN_PKG}/kube_codegen.sh" \
```

Comment:
I don't think this works well. Maybe, when you remove the `client-go` folder, then you perform `make generate`, client-go codes wouldn't be generated. 

I think we must use the following new command:

https://github.com/kubernetes/code-generator/blob/e4611069dfb4b0c04c7751afae1b9fef64828964/kube_codegen.sh#L381-L422

Reply:
Yeah, we probably can use this as an example: https://github.com/kubernetes/sample-controller/blob/master/hack/update-codegen.sh